### PR TITLE
chore: add partial close position on nibi perp

### PR DESCRIPTION
**Pull request by CalicoNino**
Adding the MsgPartialClose from protojs onto nibijs to be used in the frontend

### CODE REVIEW

**Comment by cgilbe27** [comment]
You might want to pull latest protojs from nibiru block chain

File: yarn.lock
Comment (cgilbe27) [inline yarn.lock:1]: Unnecessary yarn.lock change?

Comment (CalicoNino) [reply yarn.lock:1]: I thought of upgrading the protojs package in nibijs to v0.21.13

Comment (cgilbe27) [reply yarn.lock:1]: Oops, ignore me

Comment (cgilbe27) [reply yarn.lock:1]: Still a very large diff with one package change, on the right node version?

Comment (CalicoNino) [reply yarn.lock:1]: yeuup

**Comment by CalicoNino** [comment]
@cgilbe27 updated the proto from nibiru chain 0.21.9, looks like no changes

**Review by cgilbe27 — APPROVED** [review]

**Pull request merged**